lib/utils.js

```diff
@@ -115,6 +115,25 @@ var Utils = module.exports = {
       options.where = Utils.mapWhereFieldNames(options.where, Model);
     }
 
+    if (Array.isArray(options.order)) {
+      options.order.forEach(function(oGroup) {
+        var OrderModel, attr;
+        if (Array.isArray(oGroup)) {
+          OrderModel = Model;
+          attr = oGroup[oGroup.length - 2];
```

Comment:
order can be defined without a direction where this will fail.
Please add fix + test for case `[Model, field]` as opposed to `[Model, field, DESC|ASC]`